src/Components/List/List.css

```diff
@@ -85,3 +85,34 @@
 .resetButton:hover {
   cursor: pointer;
 }
+
+/* Issue #12 css */
+.inactive {
+  background-color: rgb(165, 164, 164);
```

Comment:
I like the idea of using grey as the colour for inactive items, but when an inactive item is marked as purchased, the label is disabled and turns a similar shade of grey, and can be hard to see. Maybe we change the disabled text colour to be a different colour or font weight? Or alternatively change the background to be lighter?

Reply:
Thanks for noticing this! We made the gray lighter and the strikeout darker. We also went with the badge idea so the color isn't covering the whole word.